core/src/components/chip/chip.tsx

```diff
@@ -1,12 +1,12 @@
 import { Component, Host, h, Event, EventEmitter, Prop } from '@stencil/core';
 
 @Component({
-  tag: 'sdds-chip',
-  styleUrl: 'sdds-chip.scss',
+  tag: 'tds-chip',
+  styleUrl: 'chip.scss',
   shadow: false,
   scoped: true,
 })
-export class SddsChip {
+export class TdsChip {
   /** Type of Chip, depends on usage */
```

Comment:
Same as in Button, so a small typo or change in one of these comments to it triggers Stencil's readme recreation so it will rename top of the readme to the tds-chip and not sdds-chip.

Reply:
Actually did not work, so had to fix it manually.